packages/vue-dot/cypress/tests/CopyBtn.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { mount } from '@cypress/vue';
+import { vuetify } from './vuetify';
+// import { VApp } from 'vuetify/lib/components';
+import CopyBtn from '../../src/elements/CopyBtn';
+
+describe('Testing CopyBtn', () => {
+
+	beforeEach(() => {
+		mount(CopyBtn,{
+		vuetify
+		});
+	});
+
+	it('Verify if is link correcly', () => {
+		expect(true).to.equal(true);
+	});
+
+	it('Button check and click', () => {
+		cy.document().get('button').should('exist').click();
+	});
+  });
```

Comment:
```suggestion
describe('CopyBtn', () => {
	beforeEach(() => {
		mount(CopyBtn, {
			vuetify
		});
	});

	it('copies the text to the clipboard', () => {
		cy.document().get('button').should('exist').click();
	});
});
```
Il faudrait également passer des propriétés au composant comme le texte à copier et vérifier qu'il est bien copié